cli/rbd/config/config_global.py

```diff
@@ -0,0 +1,99 @@
+from copy import deepcopy
+
+from cli.utilities.utils import build_cmd_args
+
+
+class Global:
+    """
+    This module provides CLI interface to manage the configuration override.
+    """
+
+    def __init__(self, nodes, base_cmd):
+        self.base_cmd = base_cmd + " global"
+
+    def get_(self, **kw):
+        """
+        This method is used to get a global-level configuration override.
+        Args:
+          kw(dict): Key/value pairs that needs to be provided to the installer
+          Example:
+            Supported keys:
+              config_entity(str): This is a global, client or client id.
+              key(str): This is the config key.
+
+        Returns:
+          Dict(str)
+            A mapping of host strings to the given task’s return value for that host’s execution run
+        """
+        kw = kw.get("kw")
+        config_entity = kw.get("config_entity")
+        key = kw.get("key")
+        cmd = self.base_cmd + f" get {config_entity} {key}"
+
+        return self.execute(cmd=cmd)
+
+    def list_(self, **kw):
```

Comment:
We decided to pass mandatory parameters as method signature and optional parameters through `kw`